server/routes/explore/api.py

```diff
@@ -160,6 +160,7 @@ def _fulfill_with_insight_ctx(request: Dict, debug_logs: Dict,
                               counters: ctr.Counters) -> Dict:
   insight_ctx = request.get_json()
   test = request.args.get(Params.TEST.value, '')
+  i18n = request.args.get(Params.I18N.value, '')
```

Comment:
This flow is for `/fulfill`, and it never sets the lang.  So I think if we want to support i18n for this flow, we should pass in `i18n_lang` as param.  In this PR, maybe drop changes to this file, and have a TODO to support i18n for `/fulfill` API?

Reply:
Done.